src/pages/docs/runbooks/config-as-code-runbooks.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-11-24
+modDate: 2024-11-24
+title: Configuration as Code runbooks
+description: Details about using configuration as code with runbooks.
+navOrder: 20
+icon: fa-brands fa-git-alt
+---
+
+- cac runbooks allows you to version control your runbook process alongside your project code - just as you can already do with your cac deployment process, deployment settings and non-sensitive variables.
+- Cac runbooks is optional. On existing projects and non-VCS projects, you can continue using runbooks just as you always have.
+
+## How to convert
+
+New version controlled projects will have CaC runbooks automatically when the the project converts to version controlled. (link to conversion info in main cac docs)
+
+:::div{.info}
+Converting a project to use Git runbooks is a one-way change. Once you convert a project, you **cannot** convert it back. Please make sure you want to do this, and consider cloning your project to test how it works, so you know what to expect before converting important projects.
+:::
+
+Projects which converted to Git before the introduction of config-as-code runbooks can be easily updated. You can migrate an existing version controlled project to use config-as-code runbooks by clicking on the 'Store Runbooks in Git' banner at the top of the **Runbooks** page of your project.
+
+- You can check your runbooks are cac by checking the runbooks page for the branch selector or your repository for the runbooks folder. 
+
+
+## How are runbooks different with CaC?
```

Comment:
Do we need to mention anything here about when you run a runbook it always uses the tip of the selected branch?